api/types/types.go

```diff
@@ -451,7 +452,7 @@ type NetworkCreate struct {
 	CheckDuplicate bool                     `json:",omitempty"`
 	Driver         string                   // Driver is the driver-name used to create the network (e.g. `bridge`, `overlay`)
 	Scope          string                   // Scope describes the level at which the network exists (e.g. `swarm` for cluster-wide or `local` for machine level).
-	EnableIPv6     bool                     // EnableIPv6 represents whether to enable IPv6.
+	EnableIPv6     optional.Option[bool]    // EnableIPv6 represents whether to enable IPv6.
```

Comment:
```suggestion
	EnableIPv6     optional.Option[bool]    `json:",omitempty"` // EnableIPv6 represents whether to enable IPv6.
```
to suppress marhsalling `{"EnableIPv6": null}`.